easyrsa3/easyrsa

```diff
@@ -674,6 +680,79 @@ current CA keypair. If you intended to start a new CA, run init-pki first."
             fi
         fi
 	fi
+
+	# Generate the CA with AES256
+
+	# => BEGIN SSL lib version
+	case "$osslv_major" in
+
+	# BEGIN SSL V3
+	3)
+		# OpenSSL v3: '-nodes' is deprecate, use '-noenc'
+		unset -v no_password; [ -z "$nopass" ] || no_password='-noenc'
+
+		# Generate CA
+		case "$EASYRSA_ALGO" in
+		rsa)
+			# OpenSSL v3: 'genrsa' is deprecate, use 'req'
```

Comment:
You are certainly right that genrsa is deprecated. However I still don't understand why you went with req -newkey instead of just replacing genrsa with genpkey? Having only one shared req call like in the OpenSSL v1 seems to make for less complex code.